includes/Context.php

```diff
@@ -165,6 +165,19 @@ public function is_network_mode() {
 		return $network && $site->domain === $network->domain;
 	}
 
+	/**
+	 * Gets the cannonical "home" URL.
+	 *
+	 * Returns the value from the `"googlesitekit_canonical_home_url"` filter.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string Cannonical home URL.
+	 */
+	public function get_canonical_home_url() {
+		return apply_filters( 'googlesitekit_canonical_home_url', home_url() );
```

Comment:
Every filter should have documentation:
```suggestion
		/**
		 * Filters the canonical home URL considered by Site Kit.
		 *
		 * Typically this is okay to be the unmodified `home_url()`, but certain plugins (e.g. multilingual plugins)
		 * that dynamically modify that value based on context can use this filter to ensure that the URL considered
		 * by Site Kit remains stable.
		 *
		 * @since n.e.x.t
		 *
		 * @param string $home_url The value of `home_url()`.
		 */
		return apply_filters( 'googlesitekit_canonical_home_url', home_url() );
```